### PR TITLE
Create docker.yaml github workflow

### PR DESCRIPTION
**Pull request by julian9499**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:10 AS builder
+FROM node:12 AS builder
```
Comment (martijnjanssen) [inline Dockerfile:1]: Why only bump this version and not the second `FROM`?

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: Docker
+on:
+  push:
+  schedule:
+  - cron: '54 2 2 * *'
```
Comment (martijnjanssen) [inline .github/workflows/docker.yaml:5]: Every 2nd day of the month on 02:54? Am I reading this right?

File: Dockerfile (hunk 1 above)
Comment (julian9499) [reply Dockerfile:1]: I only did this to manage build compatibility, but you are right I would suggest we update to node 12 across all the builders

Comment (julian9499) [reply Dockerfile:1]: dockefiles*

File: .github/workflows/docker.yaml (hunk 2 above)
Comment (julian9499) [reply .github/workflows/docker.yaml:5]: Leftover from another repo mb

Comment (bramvankooten) [reply .github/workflows/docker.yaml]: Guess Mark thought of that one 😆

File: .github/workflows/docker.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        with:
+          images: ghcr.io/areafiftylan/lancie-frontend
+          tags: type=sha, prefix={{date 'YYYYMMDD'}}-
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          push: ${{ github.ref == 'refs/heads/master' }}
```
Comment (martijnjanssen) [inline .github/workflows/docker.yaml:34]: Push should only be on the `live` branch for this repo.

**Review by bramvankooten — APPROVED** [review]

**Pull request merged**